hw_diag/utilities/balena_supervisor.py

```diff
@@ -56,11 +56,24 @@ def get_device_status(self, key_to_return) -> str:
         LOGGER.info("Retrieving device status using Balena supervisor.")
 
         response = self._make_request('GET', '/v2/state/status')
-        if response is None or response.ok is False:
-            LOGGER.error("Device status request failed.")
-            raise Exception("Device status request failed")
+
+        error_msg = ""
+        if response is None:
+            error_msg = "Device status request failed. No response recieved."
+
+        elif response.ok is False:
+            error_msg = "Device status request failed. Got non-OK response: " + \
+                  f"{response.status_code} {response.content}"
+
+        if error_msg:
+            LOGGER.error(error_msg)
```

Comment:
Should this be warning level? We expect invalid responses occasionally. We are in the process of making Sentry only listen for CRITICAL level logs but when we eventually switch back to ERROR level, this shouldn't trigger Sentry.

Reply:
Makes sense. Unfortunately I merged the branch before seeing this comment. Have updated it and pushed the branch a fresh now.

Reply:
@marvinmarnold - the new PR with this change is [here](https://github.com/NebraLtd/hm-diag/pull/375)